### PR TITLE
Implement human readable output for `security-status` (SC-286)

### PR DESCRIPTION
**Pull request by renanrodrigo**
Bring the output from `ubuntu-security-status` to the client, to the `security-status` subcommand.
Implement the human readable output based on what is currently presented in `ubuntu-security-status`.

## Desired commit type
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist
 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Comment by renanrodrigo** [comment]
hmmm there's a test to check, doesn't look like it's flaky, I'll see that

File: features/security_status.feature
Comment (lucasmoura) [inline features/security_status.feature:49]: Let's add the schema check step here

File: features/security_status.feature
Comment (lucasmoura) [inline features/security_status.feature:92]: We can remove this bit

File: features/security_status.feature
Comment (lucasmoura) [inline features/security_status.feature:188]: I think we should change this to `pro enable esm-infra` right ?

File: uaclient/security_status.py
Comment (lucasmoura) [inline uaclient/security_status.py:431]: This bit is really similar to the `list_unavailable_packages`, maybe we can isolate it in a helper function

File: uaclient/security_status.py
Comment (lucasmoura) [inline uaclient/security_status.py:384]: Let's pass the `cfg` object when creating new entitlement objects, otherwise we will create another `UAConfig` instance each time

File: uaclient/security_status.py
Comment (lucasmoura) [inline uaclient/security_status.py:330]: Should we be defensive here and use `get` instead` ?

File: uaclient/security_status.py
Comment (lucasmoura) [inline uaclient/security_status.py:348]: let's change it to  `pro enable`

File: uaclient/security_status.py
Comment (lucasmoura) [inline uaclient/security_status.py:371]: let's change it to `pro enable`

File: features/security_status.feature
Comment (renanrodrigo) [reply features/security_status.feature:92]: sure, will take it out when ready again

**Comment by renanrodrigo** [comment]
@lucasmoura I resolved your comments as I am marking this as a draft again, and I'm doing more refactors.

**Comment by renanrodrigo** [comment]
It's there again for review. Maybe there will be message tweaks but generally that's it.

**Review by lucasmoura — APPROVED** [review]

**Review by orndorffgrant — COMMENTED** [review]
Some small nits from code review

File: uaclient/system.py
Comment (renanrodrigo) [reply uaclient/system.py:300]: Yeah, in this case we need to be aware of non-LTS releases (which don't have an `esm-eol` date)
I'm on it

Comment (renanrodrigo) [reply uaclient/system.py]: Well I considered that non-lts have the same date for `eol` and `esm-eol`, to help with consistency.

**Review by orndorffgrant — APPROVED** [review]
Thanks! I'll just wait for CI before merging

**Pull request merged**
